pkg/server/container_create.go

```diff
@@ -87,8 +87,7 @@ func (c *criService) CreateContainer(ctx context.Context, r *runtime.CreateConta
 	}
 	sandboxPid := s.Pid()
 
-	trusted := sandbox.Config.Annotations[annotations.PrivilegedSandbox] == "true"
-	containerRuntime := c.getRuntime(trusted)
+	ociRuntime := c.getOCIRuntime(hostPrivilegedSandbox(sandbox.Config))
```

Comment:
Do you expect that all the runtimes registered in the daemon are an OCI runtime?. I ask because as far I understand containerd could have different runtime plugins and some of them may not be an OCI compatible plunging (depending on `ociRuntime.Type` value).  

From my perspective, it make it more clear that the underling runtime is an OCI runtime (not obvious when I see  `io.containerd.runtime.v1.linux`), but not sure is there a plan support any other type of runtime plugin in the future.

Reply:
Good question. Let me confirm it.

Reply:
Hm. It seems that containerd could potentially support non OCI runtimes.

However, I think for cri-containerd, we do only care about OCI runtime for now. :)